readme.rst

```diff
@@ -5,61 +5,30 @@ See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-a
 
 Then go through the following sections to use this starter pack to set up your docs repository.
 
-Set up your documentation repository
+Prerequisites
 ------------------------------------
 
-You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
+Before you begin, ensure you have the following:
 
-**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
+ * A GitHub repository where you want to host your documentation. The recommended approach is to host the documentation alonside your code in a `docs` folder. But a standalone docs repo is also an option.
+ * Git and Bash installed on your system.
 
-Standalone documentation repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-To create a standalone documentation repository, clone this starter pack
-repository, `update the configuration <#configure-the-documentation>`_, and
-then commit all files to the documentation repository.
-
-You don't need to move any files, and you don't need to do any special
-configuration on Read the Docs.
+Bootstrap your documentation repository
+------------------------------------
 
-Here is one way to do this for newly-created fictional docs repository
-``canonical/alpha-docs``:
+Bootstrap your repository by following these steps:
 
-.. code-block:: none
+ * Copy [bootstrap.sh](bootstrap.sh) to your repository's root directory.
+ * Run the script: ``./boostrap.sh``
+ * Enter the installation directory when prompted. For standalone repos, enter ".". For docs alonside code, enter "docs".
```

Comment:
```suggestion
 * Enter the installation directory when prompted. For standalone repositories, enter ".". For documentation alongside code, enter "docs".
```

There's a few more cases of "repo" and "docs", which we shouldn't use imo. They'll also break the spelling check.

Reply:
Cool, updated.